### PR TITLE
Improve markdown formatting for adodbapi

### PR DESCRIPTION
**Pull request by Avasam**
The adodbapi version of #2209

- Used code blocks with language marker (which should add syntax highlighting depending on the viewer, and a copy button on GitHub)
- Ran the files through Markdownlint
  - Which standardized spacing and bare links
  
CC @vernondcole

### CODE REVIEW

**Comment by vernondcole** [comment]
This look great, and thank you for all the work!
I have added two comments in-line. One is a broken link which should be excluded.
The second is also a broken link, but it has an update to a different Microsoft page, so we can just change the link.

**Comment by Avasam** [comment]
Hi @vernondcole Thanks for the review! But I don't see your comments
I suppose you are talking about https://asp101.com/articles/john/connstring/default.asp and https://www.microsoft.com/en-us/download/404-page . Which one should I remove and which one should I replace by what new link?

File: adodbapi/quick_reference.md [hunk 1]
```diff
@@ -46,12 +46,11 @@ combinations. See
 and for more explanation:
```
Comment (vernondcole) [inline adodbapi/quick_reference.md:46]: This is a broken link and should be removed.

File: adodbapi/quick_reference.md [hunk 2]
```diff
@@ -61,7 +60,7 @@ document.](https://docs.microsoft.com/en-us/sql/connect/oledb/oledb-driver-for-s
 
 - Jet (ACCESS database) and other file datasets (like .xls and .csv) "ACE"
 Provider:
-    - <http://www.microsoft.com/en-us/download/details.aspx?id=13255> a (32
+  - <http://www.microsoft.com/en-us/download/details.aspx?id=13255> a (32
```
Comment (vernondcole) [inline adodbapi/quick_reference.md:63]: A better link would be https://learn.microsoft.com/en-us/office/troubleshoot/access/jet-odbc-driver-available-32-bit-version . This link is broken.

**Review by vernondcole — CHANGES_REQUESTED** [review]

**Comment by vernondcole** [comment]
Hmm. Github did not save them. I think my re-try may have worked better.

File: adodbapi/quick_reference.md (hunk 2 above)
Comment (Avasam) [reply adodbapi/quick_reference.md]: Done

File: adodbapi/quick_reference.md (hunk 1 above)
Comment (Avasam) [reply adodbapi/quick_reference.md]: Done

**Review by vernondcole — APPROVED** [review]
Look great. Merge at will.

**Pull request merged**